tests/conftest.py

```diff
@@ -1,96 +0,0 @@
-import os
-import tempfile
-
-import boto3
-from botocore.stub import Stubber
-from pytest import fixture
-
-from blobby import blob, client
-
-
-@fixture
-def s3():
-    s3 = boto3.resource('s3')
-    yield s3
-
-
-@fixture
-def stubber(s3):
-    with Stubber(s3.meta.client) as stubber:
```

Comment:
I think this is fine for now, though once this library becomes more developed we should investigate alternative testing strategies for the 3rd party clients.

Reply:
I think the fix there is integration (between us and `boto3`) testing, but yes definitely!